src/service/action-impl.js

```diff
@@ -200,6 +203,10 @@ export class ActionService {
         this.addChangeDetails_(event);
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'input-debounced') {
+      this.root_.addEventListener('input', debounce_(event => {
+        this.trigger(dev().assertElement(event.target), name, event);
```

Comment:
Danger! If this is a native event, the `target` and other properties of the event may be gone by the time this method is executed. Firefox in particular has issues.